README.md

```diff
@@ -194,6 +195,13 @@ lsr_role2collection.py [-h] [--namespace NAMESPACE] [--collection COLLECTION]
                      in the README that you do not want to change, you may have to change the
                      README in another way, not using this script, by using sed with a custom
                      regex.
+--collection-mapping If it is set and given namespace.collection is not default, the default
+                     collection fedora.linux_system_roles is converted to
+                     NAMESPACE.COLLECTION.
+--meta-runtime META_RUNTIME
+                     This is the path to the collection meta/runtime.yml - the default is
+                     /home/nhosoi/linux-system-roles/auto-
+                     maintenance/lsr_role2collection/runtime.yml.
```

Comment:
This doesn't look right

Reply:
??? I just copied the Usage since it was missing... If it's not a good idea, I'll drop it. :)

Reply:
I mean you have `/home/nhosoi` there - should it be `$HOME/linux-system-roles/auto-maintenance/lsr_role2collection/runtime.yml` instead?

Reply:
Ahhh. Sorry, I was blind... :p Then, I'm adding them back with $HOME...